presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSphericalGeoFunctions.java

```diff
@@ -158,9 +158,6 @@ public void testArea()
         // Empty polygon
         assertFunction("ST_Area(to_spherical_geography(ST_GeometryFromText('POLYGON EMPTY')))", DOUBLE, null);
 
-        // Invalid polygon (too few vertices)
```

Comment:
Why are we removing this test case?

Reply:
Previously, certain types of invalid geometries could be created, but would have to be checked for explicitly in functions such as these.  Now, those geometries fail on construction (I've updated the release notes to mention this).